service-monitor/src/main/java/com/yahoo/vespa/service/slobrok/SlobrokMonitorManagerImpl.java

```diff
@@ -76,6 +93,11 @@ public void applicationRemoved(ApplicationId id) {
     public void bootstrapComplete() {
     }
 
+    @Override
+    public void deconstruct() {
+        transport.sync().shutdown().join();
```

Comment:
extra sync up front should not be needed

Reply:
```suggestion
        transport.shutdown().join();
```